tests/test_message.py

```diff
@@ -15,7 +15,8 @@ def betamax_init(self):
     @betamax()
     def test_get_comment_replies(self):
         comment_reply = next(self.r.get_comment_replies(limit=1))
-        comment_parent = self.r.get_info(thing_id=comment_reply.parent_id)
+        info_id = comment_reply.parent_id
+        comment_parent = next(self.r.get_info(thing_id=info_id))
```

Comment:
It'd be nice if when `thing_id` only contains a single item, that `get_info` still returns that one item. Then we don't break this part of the behavior.